array.c

```diff
@@ -7843,13 +7843,22 @@ rb_ary_deconstruct(VALUE ary)
  *
  *  == Creating Arrays
  *
- *  A new array can be created by using the literal constructor
- *  <code>[]</code>.  Arrays can contain different types of objects.  For
+ *  You can create an \Array object explicitly with:
+ *
+ *  - Global method {Array}[Kernel.html#method-i-Array].
+ *  - A {array literal}[doc/syntax/literals_rdoc.html#label-Array+Literals].
+ *
+ *  Many Ruby methods return \Array objects, including:
+ *
+ *  - Instance method +to_a+, available in some Core and Standard Library classes.
+ *  - Instance method +to_ary+, available in fewer classes.
+ *
+ *  An \Array can contain different types of objects.  For
  *  example, the array below contains an Integer, a String and a Float:
  *
  *     ary = [1, "two", 3.0] #=> [1, "two", 3.0]
  *
- *  An array can also be created by explicitly calling Array.new with zero, one
+ *  An array can also be created by calling Array.new with zero, one
```

Comment:
I don't think there are implicit callers of `Array.new`, so you can drop the `explicitly`.

Reply:
Already dropped?